domain/src/test/java/org/oppia/android/domain/exploration/ExplorationProgressControllerTest.kt

```diff
@@ -3407,6 +3435,13 @@ class ExplorationProgressControllerTest {
     playThroughPrototypeState6AndMoveToNextState()
   }
 
+  private fun clickContinueOnFlashbackConfirmationDialog(stateName: String): EphemeralState {
```

Comment:
```suggestion
  private fun moveToFlashbackState(stateName: String): EphemeralState {
```

Is more descriptive.

Reply:
Thanks, Done.